action-mongo-atlas-nuke.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+# usage: python action-mongo-atlas-nuke.py
+# export MONGODB_ATLAS_PRIVATE_KEY, MONGODB_ATLAS_PUBLIC_KEY as env variable
+#########################################################################################################
+# coding=utf-8
+import os
+import requests
+import json
+from requests.auth import HTTPDigestAuth
+from retry import retry
+from typing import List
+
+
+def get_all_projects(public: str, private: str, url: str) -> List[str]:
+    project_url = url + "/groups"
+    response = requests.request("GET", project_url, auth=HTTPDigestAuth(public, private))
+    project_list = json.loads(response.content)
+    id_list = []
+    for p in project_list['results']:
+        id_list.append(p['id'])
+    return id_list
```

Comment:
I think this can be simplified to something like:
`return [p['id'] for p in project_list['results']]`

Reply:
Up to you. But for your info: https://trinket.io/python/a4dad25c2b